content/en/cloudcraft/api/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: API Reference
+type: documentation
```

Comment:
do we want to disable the toc for cloudcraft api docs?  (dd api docs dont have toc)